apps/epic-web/package.json

```diff
@@ -10,6 +10,7 @@
     "db:studio": "prisma studio",
     "dev:stripe": "stripe listen --forward-to localhost:3021/api/skill/webhook/stripe",
     "dev": "next dev -p 3021",
+    "dev:email": "email dev --dir ./src/emails",
```

Comment:
I had to make adjustments to `./react-email/package.json` after this ran removing all the workspace deps, but worked great after that